persistence-modules/hibernate-exceptions/src/main/java/com/baeldung/hibernate/unknownentityexception/Customer.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.hibernate.unknownentityexception;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "customer")
+public class Customer {
```

Comment:
Please apply the Baeldung formatter to all files, you can find it here:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij